templates/engage/fr/esm_1404.html

```diff
@@ -0,0 +1,102 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Le support d’Ubuntu 14.04 LTS est maintenant passé à ESM (Extended Maintenance Support){% endblock %}
```

Comment:
We should use the new metadata include https://github.com/canonical-web-and-design/www.ubuntu.com/pull/5255